src/main/java/com/gtnewhorizons/angelica/transform/compat/transformers/specific/StacksOnStacksTransformer.java

```diff
@@ -64,8 +62,18 @@ private InsnList injectPileRenderInitializer(MethodNode mn, String type) {
         list.add(new VarInsnNode(Opcodes.ALOAD, 0));
         list.add(new TypeInsnNode(Opcodes.NEW, "com/tierzero/stacksonstacks/client/render/PileRender" + type));
         list.add(new InsnNode(Opcodes.DUP));
-        list.add(new MethodInsnNode(Opcodes.INVOKESPECIAL, "com/tierzero/stacksonstacks/client/render/PileRender" + type, "<init>", "()V"));
-        list.add(new FieldInsnNode(Opcodes.PUTFIELD, "com/tierzero/stacksonstacks/client/render/RenderTilePile", type.toLowerCase() + "Render", "Lcom/tierzero/stacksonstacks/client/render/PileRender;"));
+        list.add(
```

Comment:
the mn parameter from the method isn't used, you can remove it and make the method static